lighthouse-core/scripts/generate-bundlephobia-database.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+'use strict';
+
+/* eslint-disable no-console */
+
+// TODO(saavan) These type definitions are incorrect and need to be fixed.
+/** @typedef {{gzip: number, name: string, version: string | number, repository: string}} BundlePhobiaLibrary */
+
+const fs = require('fs');
+const getPackageVersionList = require('bundle-phobia-cli').fetchPackageStats.getPackageVersionList;
+const fetchPackageStats = require('bundle-phobia-cli').fetchPackageStats.fetchPackageStats;
+
+const databasePath = '../audits/byte-efficiency/bundlephobia-database.json';
+
+/** @type {Record<string, string[]>} */
+const suggestionsJSON = require('../audits/byte-efficiency/library-suggestions.js').suggestions;
+/** @type string[] */
+/* eslint-disable-next-line max-len */
+const librarySuggestions = Object.keys(suggestionsJSON).map(s => suggestionsJSON[s].concat(s)).flat();
```

Comment:
😍 but unfortunately `.flat` starts at node 11 and we're committed to 10.x for now :/ not a big deal since it's in a script not user-run code, but it'd be nice to stay consistent

maybe...?

```suggestion
const librarySuggestions = [].concat(...Object.values(suggestionsJSON));
const allLibraryNames = librarySuggestions.concat(Object.keys(suggestionsJSON));
```